test-data/unit/check-classes.test

```diff
@@ -5436,7 +5436,7 @@ main:8:6: error: Type argument "builtins.int" of "G" must be a subtype of "built
 [case testExtremeForwardReferencing]
 from typing import TypeVar, Generic
 
-T = TypeVar('T')
+T = TypeVar('T', covariant=True)
```

Comment:
Previous behavior was incorrect. With this change `covariant` is needed to keep the old behavior